bindata/network/kuryr/004-config.yaml

```diff
@@ -0,0 +1,462 @@
+# More info about the template: https://docs.openstack.org/kuryr-kubernetes/latest/installation/containerized.html#generating-kuryr-resource-definitions-for-kubernetes
+
+# TODO: pod_subnets_driver (Rocky)
+#       enabled_drivers    (Rocky)
+#       vif_pool_driver
+#       HA                 (Rocky)
+#       [cni_health_server] (Rocky)
+
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kuryr-config
+  namespace: kuryr
+data:
+  kuryr.conf: |+
```

Comment:
Any reason to keep these commented-out lines? Not sure what purpose they serve.

Reply:
This is standard way of providing OpenStack services config template. It was generated by OpenStack tools and operators are used to see it that way. We can discuss squeezing it, but for now I'd rather keep it like that through exploratory period.